content/en/platform/corda/5.0/developing-applications/getting-started/utxo-advanced-ledger-extensions/advanced-design.md

```diff
@@ -0,0 +1,334 @@
+---
+date: '2023-06-01'
+title: "Advanced Contract Design"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-utxo-ledger-extensions-advanced-design
+    parent: corda5-utxo-advanced-ledger-extensions
+    weight: 4600
+section_menu: corda5
+---
+
+# Advanced Contract Design
+
+All the contract design issues that have been highlighted above are implemented by the Corda 5 Advanced UTXO Extensions library, and are included in all the specific implementations; for example, chainable, fungible and identifiable contracts.
+
```

Comment:
I think this bit belongs to the design discussion you put in the `commands-example.md`, but which should be here.
The rest of this file isn't really about advanced design but in detail description of the different states/contracts plus examples.

Reply:
I've moved that section to the other file and updated it with a link accordingly.